Dockerfile

```
@@ -1,10 +1,23 @@
+   
 WORKDIR /app
-
+    
 COPY package.json pnpm-lock.yaml checkNodeVersion.js ./
-
+    
 RUN pnpm install
+    
+   
+# ---------- Build Stage ----------
+FROM base AS build
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)